RELEASE.md

```diff
@@ -159,6 +162,7 @@ Currently, there are only 2 activities we use rever for, (1) aggregating the aut
 2. Run `rever --activities authors <VERSION>`:
 
     > **Note:** <!-- GH doesn't support nested admonitions, see https://github.com/orgs/community/discussions/16925 -->
+    > `<VERSION>` means `YY.M.[$patch_number]` through out this document (e.g. 24.9.1).</br>
```

Comment:
We already have a very similar note at the very top of this file. Can this be combined?